libcouchbase: error build

## Issue description
error build:
```
build/source/build/bin/Development/check-all
build/source/build/bin/Development/check-all
Unable to find executable: /build/source/build/bin/check-all
 9/10 Test #10: check-libevent-sock-tests ........***Not Run   0.00 sec
10/10 Test  #1: BUILD-TESTS ......................   Passed   66.32 sec

10% tests passed, 9 tests failed out of 10

Total Test time (real) =  66.32 sec

The following tests FAILED:
          2 - check-select-nonio-tests (Not Run)
          3 - check-select-rdb-tests (Not Run)
          4 - check-select-vbucket-tests (Not Run)
          5 - check-select-mc-tests (Not Run)
          6 - check-select-htparse-tests (Not Run)
          7 - check-select-unit-tests (Not Run)
          8 - check-select-sock-tests (Not Run)
          9 - check-libevent-unit-tests (Not Run)
         10 - check-libevent-sock-tests (Not Run)
Errors while running CTest
make: *** [Makefile:107: test] Error 8
```

Commit edc3559c1ae1cd1edeff11af52ae768359b38ba6 - normal build, 3e950d5 - error.

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.7, NixOS, 19.03pre161768.561959bab88 (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2pre6526_9f99d624`
 - channels(root): `"nixos-19.03pre161768.561959bab88"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

--- a/pkgs/development/libraries/libcouchbase/default.nix
+++ b/pkgs/development/libraries/libcouchbase/default.nix
@@ -16,6 +16,9 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake pkgconfig ];
   buildInputs = [ libevent openssl ];
 
+  # Running tests in parallel does not work
+  enableParallelChecking = false;
+
   doCheck = !stdenv.isDarwin;
 
   meta = with stdenv.lib; {

